jormungandr-integration-tests/src/common/jcli_wrapper/mod.rs

```diff
@@ -285,18 +283,29 @@ pub fn assert_rest_get_next_block_id(block_id: &str, id_count: &i32, host: &str)
 
 pub fn assert_transaction_in_block(transaction_message: &str, host: &str) -> Hash {
     let fragment_id = assert_post_transaction(&transaction_message, &host);
-    wait_until_transaction_processed(fragment_id, &host);
+    wait_until_transaction_processed(fragment_id, &host, &Default::default());
```

Comment:
The `Default::default` is a nice trick, but here it obscures exactly what type is defaulted